src/widgets/mainwindow.h

```diff
@@ -43,10 +43,11 @@ public slots:
     //! Update the Tarsnap version number, and store it in the settings.
     void updateTarsnapVersion(QString versionString);
     void notificationRaise();
-    //! Display stop tasks dialog. Also used when quitting the application 
-    //! while active or background tasks are queued.
-    void displayStopTasks(bool backupTaskRunning, int runningTasks,
-                          int queuedTasks);
+    //! Prompt user to clarify whether to stop background tasks; if so,
+    //! quits the app. Also used when quitting the application while active
```

Comment:
This is a bit incorrect. Since the dialog's principal purpose is the stop tasks action, which is invoked either via the Ctrl^c or click on the busy icon. Incidentally it is also used when trying to quit the application with active and/or queued actions. Thus the comment could be a tad improved.